integrations/aws/main.py

```diff
@@ -33,21 +37,49 @@
 )
 
 
+async def _handle_global_resource_resync(
+    kind: str,
+    credentials: AwsCredentials,
+    current_session: aioboto3.Session,
+    handle_exceptions: bool = True,
+) -> ASYNC_GENERATOR_RESYNC_TYPE:
+    try:
+        async for batch in resync_cloudcontrol(kind, current_session):
+            yield batch
+    except Exception as e:
+        if is_access_denied_exception(e):
+            if handle_exceptions:
+                logger.info(f"Trying to resync {kind} in all regions until success")
+                async for session in credentials.create_session_for_each_region():
+                    s = await session
+                    try:
+                        async for batch in resync_cloudcontrol(kind, s):
+                            yield batch
+                        break
+                    except Exception as e:
+                        continue
+
```

Comment:
what if there is any other error? what will we do with it? seems like you totally silence it

Reply:
The error message is inside `resync_cloudcontrol`